.github/workflows/schedule-dependencies.yml

```diff
@@ -1,8 +1,10 @@
 name: Cron Test Dependencies
 
 on:
+  workflow_dispatch:
```

Comment:
This is highly unnecessary, but maybe come useful for testing --pre on a specific branch before merging.
~We~ I don't have another way of running it